x-pack/plugins/security_solution/public/detection_engine/rule_management/logic/coverage_overview/models/rule_data.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export interface CoverageOverviewRuleData {
```

Comment:
Could we reuse the `CoverageOverviewRuleData` from the common folder?

Reply:
One defined in `common/.../response_schema.ts` is a DTO and another one here in `rule_data.ts` is a domain model.

Generally speaking we can reuse it but it will lead to excess `id` added in the API response which I'd like to avoid.